projects/wipp-frontend-app/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,22 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { PageNotFoundComponent } from 'wipp-frontend-lib';
+
+
+const appRoutes: Routes = [
+    { path: '',   redirectTo: '/images-collections', pathMatch: 'full' },
+    { path: '**', component: PageNotFoundComponent }
+  ];
+  
+  @NgModule({
+    imports: [
+      RouterModule.forRoot(
+        appRoutes,
+        { enableTracing: true } // <-- debugging purposes only
```

Comment:
If this is for debugging only, can we configure it via the environments file?

Reply:
this is used to be part of the original code, we could move it to the environments file though